### PR TITLE
Added SessionStack plugin documentation

### PR DESCRIPTION
**Pull request by lsnickolov**

### CODE REVIEW

File: docs/integrations/sessionstack.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+SessionStack
+============
+
+SessionStack error session replay plugin for Sentry
+```````````````````````````````````````````````````
+
+To integrate the SessionStack player within your Sentry error reports, follow the steps below:
+Add your web app to your SessionStack account
+
+.. image:: http://blog.sessionstack.com/wp-content/uploads/2017/04/create_website_opened.png
```
Comment (dcramer) [inline docs/integrations/sessionstack.rst]: we might want to upload this as part of the docs site to ensure availability, but i wont block on that

Comment (mattrobenolt) [inline docs/integrations/sessionstack.rst]: This actually won't work since it's an http asset being served on an https asset. We need to vendor it.

**Comment by dcramer** [comment]
I'll go ahead and vendor it

**Comment by dcramer** [comment]
https://github.com/getsentry/sentry-docs/pull/136

**Pull request merged**